internal/provider/provider_test.go

```diff
@@ -2,24 +2,3 @@
 // SPDX-License-Identifier: MPL-2.0
 
 package provider
-
-import (
-	"testing"
-
-	"github.com/hashicorp/terraform-plugin-framework/providerserver"
-	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
-)
-
-// testAccProtoV6ProviderFactories are used to instantiate a provider during
-// acceptance testing. The factory function will be invoked for every Terraform
-// CLI command executed to create a provider server to which the CLI can
-// reattach.
-var testAccProtoV6ProviderFactories = map[string]func() (tfprotov6.ProviderServer, error){
```

Comment:
i will add all this test stuff back in in my other PR, but since it's unused here, we are getting that lint warning.